### PR TITLE
Fixed Spelling mistake

### LINKED ISSUE #90714: Misspelt @interface name: GodotApplicalitionDelegate in platform/ios/godot_app_delegate.h
### Tested versions

master branch

### Issue description

The **Application** part in the `@interface` name `GodotApplicalitionDelegate` is misspelt.

### Files referenced in

https://github.com/godotengine/godot/blob/master/platform/ios/godot_app_delegate.h
https://github.com/godotengine/godot/blob/master/platform/ios/godot_app_delegate.m
https://github.com/godotengine/godot/blob/master/platform/ios/main.m

### PR DESCRIPTION
**Pull request by zolsonchamp**
* *Bugsquad edit, closes: https://github.com/godotengine/godot/issues/90714*

### CODE REVIEW

**Comment by zolsonchamp** [comment]
Solution to this raised issue:
https://github.com/godotengine/godot/issues/90714

**Comment by Sauermann** [comment]
Please make sure, that Godot compiles on iOS with your changes.
It looks like the interface is misspelled in multiple places.

File: platform/ios/main.m [hunk 1]
```diff
@@ -46,7 +46,7 @@ int main(int argc, char *argv[]) {
 	gargv = argv;
 
 	@autoreleasepool {
-		NSString *className = NSStringFromClass([GodotApplicalitionDelegate class]);
+		NSString *className = NSStringFromClass([GodotApplicationDelegate class]);
```
Comment (bruvzg) [inline platform/ios/main.m:49]: The same should be changed in the [platform/ios/godot_app_delegate.h](https://github.com/godotengine/godot/blob/master/platform/ios/godot_app_delegate.h) and [platform/ios/godot_app_delegate.m](https://github.com/godotengine/godot/blob/master/platform/ios/godot_app_delegate.m).

**Comment by akien-mga** [comment]
Closing as the PR is abandoned and not functional.

**Pull request closed without merging**